### PR TITLE
standardize media filenames across scrapers; bump ruby version

### PR DESCRIPTION
**Pull request by oneroyalace**
This PR, along with https://github.com/TechAndCheck/hypatia/pull/46, https://github.com/cguess/zorki/pull/15, and https://github.com/TechAndCheck/forki/pull/20, standardizes media filenames across scrapers. It also bumps the gem's ruby version. 

# Testing instructions
`rake test`

### CODE REVIEW

File: lib/youtubearchiver/video.rb [hunk 1]
```diff
@@ -64,7 +64,7 @@ def parse(video_hash)
     def download_video
       return if @live
 
-      filename = "#{YoutubeArchiver.temp_storage_location}/#{SecureRandom.uuid}.mp4"
```
Comment (cguess) [inline lib/youtubearchiver/video.rb:67]: In the other changed line, you determine the extension, but here it's just `.mp4` Would it be better to determine in both places?

**Review by cguess — CHANGES_REQUESTED** [review]
One question, then we're good.

File: lib/youtubearchiver/video.rb (hunk 1 above)
Comment (oneroyalace) [reply lib/youtubearchiver/video.rb:67]: Oops, definitely. Agree.

Comment (oneroyalace) [reply lib/youtubearchiver/video.rb:67]: Oh, well YouTube _may_ be a special case since we specify the filetype we wish to download in `yt-dlp`

**Review by cguess — APPROVED** [review]

**Pull request merged**